solidity/contracts/LinkEx.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity 0.4.24;
+
+/**
+ * @title The LINK exchange contract
+ */
+contract LinkEx {
+
+  uint256 private historicRate;
+  uint256 private rate;
+  uint256 private rateHeight;
+
+  function currentRate() public view returns (uint256) {
+    if (isFutureBlock()) {
+      return rate;
+    }
+    return historicRate;
+  }
+
+  function update(uint256 _rate) public {
+    if (isFutureBlock()) {
```

Comment:
What is the scenario that this check is guarding against? I think we could cut it.